src/apdu.c

```diff
@@ -210,6 +210,14 @@ __attribute__((noreturn)) void main_loop(struct app_handlers const *const app_ha
             CATCH(EXCEPTION_IO_RESET) {
                 THROW(EXCEPTION_IO_RESET);
             }
+            //CATCH(EXC_PARSE_ERROR) {
+            //    clear_apdu_globals(); // IMPORTANT: Application state must not persist through errors
+            //    THROW(EXCEPTION_IO_RESET);
+            //}
+            //CATCH(EXC_REJECT) {
+            //    clear_apdu_globals(); // IMPORTANT: Application state must not persist through errors
+            //    THROW(EXCEPTION_IO_RESET);
+            //}
```

Comment:
This is added here to sort of emphasize that this code _shouldn't_ exist, right? Or phrased another way, this comment block says "we know we are not catching these, and that is expected"?

Reply:
It's gone now!